native-schema-registry/python/PyGsrSerDe/AvroSerDe.py

```diff
@@ -0,0 +1,42 @@
+import fastavro
+from io import BytesIO
+
+
+class AvroSerializer:
+
+    def parse_schema(self, schema):
+        return fastavro.parse_schema(schema)
+
+    def serialize(self, data: dict, schema: dict):
+        if data is None:
+            return None
+        fo = BytesIO()
+        fastavro.writer(fo, schema, data)
+        return fo.getvalue()
+
+
+class AvroDeserializer:
+
+    def __init__(self):
+        pass
```

Comment:
Why do we need to define a constructor that does nothing? Is it possible to remove it?

Reply:
Removed, thanks